test-case/manual/check-audio-privacy-during-D3.md

```diff
@@ -0,0 +1,54 @@
+# Verify audio privacy (hard mute) functionality during D3 state
+Verify audio privacy (hard mute) functionality in Linux in D3 state
+
+## Preconditions
+1. The RVP should have active audio endpoints.
+2. The RVP should have a functional hard mute switch or toggle.
+3. Enable BIOS settings:
+   * Microphone Privacy Mode = "HW managed Microphone Privacy"
+   * Enable corresponding codec options, e.g., SNDW#3 [x] for SoundWire.
+   * Enable DMIC option, e.g., DMIC [x] for DMIC
+4. Make sure UCM is up to date
+   * Copy the ucm and ucm2 trees to the alsa-lib configuration directory (usually located in /usr/share/alsa) including symlinks
+   * Reference: https://github.com/alsa-project/alsa-ucm-conf/
+
+## Test Description
+* Verification of audio privacy (hard mute) functionality during suspension in D3 state in linux system.
+* Should not capture audio samples when the audio privacy switch is turned ON during D3 state.
+* Should capture audio samples when the audio privacy switch is turned OFF during D3 state.
+
+## Steps to Execute
+1. System is booted in the OS
+2. Capture audio via 3.5mm jack ports:
+    ```bash
+    arecord -Dhw:0,0 -c 2 -r 48000 -f S16_LE /tmp/test_sample_mute_during_d3.wav -vvv
+    ```
+3. Enter D3 state via command:
+    ```bash
+    sudo rtcwake -m mem --seconds 30
+    ```
+4. Turn ON the audio privacy switch. Led indicator should be turned ON.
+5. Exit D3 state, check on the file if audio was muted in D3 state
```

Comment:
```suggestion
5. Exit D3 state, listen to the file and check that audio was 1. actually recorded before D3, 2. muted in D3 state
```

If there is no audio in 1. (which is very common because Linux audio configuration is incredibly difficult to get right), then 2. does not prove anything. So, 1. is critical.